index.html

```diff
@@ -119,7 +119,7 @@ <h3 class="worktitle">My Recent Work</h3>
                 <img src="./pictures/myproject.jpg" alt="image of small boulder in desert, temporary image while project is completed" class="projectpic1">
                 <div class="jobdesc">
                 <p class="work1title">Multi-Post Stories</p>
```

Comment:
- [ ] This work/project section should be created dynamically using JavaScript, kindly remove the whole section from your HTML code and generate it dynamically using JavaScript